content/hi/docs/reference/tools/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: अन्य उपकरण
+reviewers:
+- divya-mohan0209
+content_type: concept
+weight: 150
+no_list: true
+---
+
+<!--Overview-->
+कुबेरनेट्स सिस्टम के साथ काम करने में आपकी सहायता के लिए कुबेरनेट्स में कई उपकरण शामिल हैं।
+
+<!--body-->
+
+## crictl
+
+[`crictl`](https://github.com/kubernetes-sigs/cri-tools) 
+{{<glossary_tooltip term_id="cri" text="CRI">}}-compatible कंटेनर रनटाइम के 
+निरीक्षण और डिबगिंग के लिए एक कमांड-लाइन इंटरफ़ेस है।
+
+## Dashboard
+
+[`डैशबोर्ड`](/docs/tasks/access-application-cluster/web-ui-dashboard/), कुबेरनेट्स का 
+वेब पे आधारित उपयोगकर्ता इंटरफ़ेस, आपको कुबेरनेट्स क्लस्टर में कंटेनरीकृत अनुप्रयोगों को तैनात करने, 
+उनकी समस्या का निवारण करने और क्लस्टर और उसके संसाधनों को प्रबंधित करने की अनुमति देता है।
+
+## Helm
+{{% thirdparty-content single="true" %}}
+
+[हेल्म](https://helm.sh/) पूर्व-कॉन्फ़िगर कुबेरनेट्स संसाधनों के पैकेजों के प्रबंधन के लिए एक उपकरण है। 
+इन पैकेजों को _हेल्म चार्ट_ के रूप में जाना जाता है।
+
+हेल्म का उपयोग करें:
+
+* कुबेरनेट्स चार्ट के रूप में पैक किए गए लोकप्रिय सॉफ़्टवेयर को ढूंढें और उपयोग करें।
+* अपने स्वयं के एप्लिकेशन को कुबेरनेट्स चार्ट के रूप में साझा करें।
+* अपने कुबेरनेट्स एप्लीकेशन के लिए प्रस्तुत करने योग्य बिल्ड बनाएं।
+* बुद्धिमानी से अपने कुबेरनेट्स मैनिफ़ेस्ट फ़ाइलों को प्रबंधित करें।
+* हेल्म पैकेजों के रिलीज़ प्रबंधित करें।
+
+## Kompose
+
+[`कॉम्पोज़`](https://github.com/kubernetes/kompose) एक उपकरण है, जो डॉकर कंपोज़ उपयोगकर्ताओं 
+को कुबेरनेट्स पर जाने में मदद करता है।
+
+कॉम्पोज़ का उपयोग करें:
+
+* डॉकर कंपोज़ फ़ाइल को कुबेरनेट्स ऑब्जेक्ट्स में अनुवाद करें।
+* स्थानीय डॉकर डेवलपमेंट से कुबेरनेट्स एप्लीकेशनों को प्रबंधित करें।
+* v1 या v2 डॉकर कंपोज़, `yaml` फ़ाइलों या
+  [वितरित एप्लिकेशनो के बंडलों](https://docs.docker.com/compose/bundles/) के माध्यम से अपने एप्लिकेशन
```

Comment:
instead of `वितरित एप्लिकेशनो के बंडलों` i think `डिस्ट्रीब्यूटेड एप्लिकेशन बंडलों `will look good.